packages/web-components/fast-foundation/src/accordion-item/accordion-item.template.ts

```diff
@@ -1,56 +1,55 @@
-import { html, ref } from "@microsoft/fast-element";
-import type { ViewTemplate } from "@microsoft/fast-element";
-import { endSlotTemplate, startSlotTemplate } from "../patterns/start-end.js";
-import type { FoundationElementTemplate } from "../foundation-element/foundation-element.js";
-import type { AccordionItem, AccordionItemOptions } from "./accordion-item.js";
+import { ElementViewTemplate, html, ref } from "@microsoft/fast-element";
+import { createEndSlotTemplate, createStartSlotTemplate } from "../patterns/start-end.js";
+import type { AccordionItemOptions, FoundationAccordionItem } from "./accordion-item.js";
 
 /**
- * The template for the {@link @microsoft/fast-foundation#(AccordionItem:class)} component.
+ * The template for the {@link @microsoft/fast-foundation#(FoundationAccordionItem:class)} component.
  * @public
  */
-export const accordionItemTemplate: FoundationElementTemplate<
-    ViewTemplate<AccordionItem>,
-    AccordionItemOptions
-> = (context, definition) => html`
-    <template class="${x => (x.expanded ? "expanded" : "")}">
-        <div
-            class="heading"
-            part="heading"
-            role="heading"
-            aria-level="${x => x.headinglevel}"
-        >
-            <button
-                class="button"
-                part="button"
-                ${ref("expandbutton")}
-                aria-expanded="${x => x.expanded}"
-                aria-controls="${x => x.id}-panel"
-                id="${x => x.id}"
-                @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
+export function createAccordionItemTemplate(
```

Comment:
Regarding your question about the naming of this function, I think this is strongest because having the verb allows it to read more like an action or function that's being called, which sometimes feels lost when defining as a `const` instead.
For whether to include "foundation" or not, without is nice because once we get to phase 5 where the template is being constructed, the same name will apply. Chances are the default fixed or constructed templates will work for a lot of people now, so while including "foundation" does make it clear, it's not as useful as it is with the renamed class export because we were _always_ aliasing there.